site/content/entry/yuzu-migration/index.md

```diff
@@ -0,0 +1,80 @@
++++
+date = "2019-10-04T23:51:00+05:30"
+title = "yuzu - The Migration"
+author = "CaptV0rt3x"
+forum = 150132
++++
+
+A very good day to you all ***yuz-ers*** out there! 
+We are currently in the process of merging both the Canary and Nightly versions of yuzu into a single version.<br>
+Jump right in to find out more about this!
```

Comment:
Can you get rid of the BR and make this part of the previous paragraph? the formatting looks weird on the preview